app/models/background.rb

```diff
@@ -44,11 +77,22 @@ def summarize
     results
   end
 
+  def pipeline_run_ids
+    # get the list of successful and most recent pipeline_runs that belong to the same samples
+    updated_pipeline_run_ids = []
+    pipeline_runs.each do |pr|
+      pr_new = pr.sample.pipeline_runs.find_by(job_status: PipelineRun::STATUS_CHECKED)
```

Comment:
Is there a difference between this and just selecting from PipelineRun.top_completed_runs based on sample_id?